src/GraphODataTemplateWriter/CodeHelpers/Java/TypeHelperJava.cs

```diff
@@ -753,6 +752,22 @@ public static string CreatePackageDefForEnum(this CustomT4Host host)
             sb.Append("\n");
             return sb.ToString();
         }
+        public static string GetPackagePrefix(this OdcmObject obj)
+        {
+            switch(obj)
+            {
+                case OdcmEnum e:
+                    return "";
```

Comment:
I made a typo in my suggestion
```suggestion
                    return "models.generated";
```